app/User.php

```diff
@@ -30,4 +30,14 @@ class User extends Authenticatable
 
     protected $test = 'testtest';
     protected $test2 = 'test2test2';
+
+    private $check_WIP = true;
+    private $check_WIP1 = true;
+    private $check_WIP2 = true;
+
+    private function testSkipWIP () {
+        $this->check_WIP1 = false;
+        $check_WIP2 = false;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "$check_WIP2" which has the same name as the field declared at line 36. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://relisonar.relipasoft.com:9000/coding_rules#rule_key=php%3AS1117)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "$check_WIP2" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://relisonar.relipasoft.com:9000/coding_rules#rule_key=php%3AS1481)